### PR TITLE
✨ Adds new option `timeoutSeconds` to inform the the timeout for webhook resources

### PR DESCRIPTION
**Pull request by yaa110**
I needed to change `timeoutSeconds` default value in a project created using `kubebuilder` and I realized that it is not possible to do that. So, I opened this PR.

### CODE REVIEW

File: pkg/webhook/parser_integration_test.go [hunk 1]
```diff
@@ -131,6 +131,35 @@ var _ = Describe("Webhook Generation From Parsing to CustomResourceDefinition",
 		Expect(err).To(MatchError("SideEffects should not be set to `Some` or `Unknown` for v1 {Mutating,Validating}WebhookConfiguration"))
 	})
```
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply pkg/webhook/parser_integration_test.go:133]: /lgtm cancel

**Comment by yaa110** [comment]
@camilamacedo86 thank you for reviewing the PR. The commit message has been amended based on your edit.

File: pkg/webhook/parser_integration_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (yaa110) [reply pkg/webhook/parser_integration_test.go:133]: in this case the default value is used for timeout (`10 secs`) and it is not an invalid value. should I add a test for testing default value?

[the remark this answers is not in the document]
Comment (camilamacedo86) [reply pkg/webhook/parser_integration_test.go:133]: That would be great. Ensure that we have more tests as possible to cover the changes is the ideal  path

**Review by camilamacedo86 — APPROVED** [review]
/hold cancel


I am ok with this one 👍 
Let's see what others think about

/lgtm

File: pkg/webhook/parser_integration_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (camilamacedo86) [reply pkg/webhook/parser_integration_test.go:133]: I think it is well covered. Thx

**Comment by sbueringer** [comment]
Thank you!

/lgtm

**Pull request merged**